tasks.py

```diff
@@ -24,7 +24,7 @@ def list(stdout=sys.stdout, filename="tasks.csv"):
         with open(filename) as tasks_file:
             reader = csv.reader(tasks_file)
             for name, completed in reader:
-                stdout.write(f"{name}{DONE if completed else ''}\n")
+                stdout.write(f"{name}{DONE if eval(completed) else ''}\n")
```

Comment:
I mentioned this on this week's homework but here it would have been better to fix this by not writing the string `"True"` and `"False"` to the CSV rather than using `eval` -- either change it to be `""` and `"x`", or here you could have done `DONE if completed == "True" else ''`